[2024-04-12T21:36:13.834+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: KingFX_ETL.Extract_Response manual__2024-04-12T21:36:09.796259+00:00 [queued]>
[2024-04-12T21:36:13.842+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: KingFX_ETL.Extract_Response manual__2024-04-12T21:36:09.796259+00:00 [queued]>
[2024-04-12T21:36:13.843+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 0
[2024-04-12T21:36:13.859+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): Extract_Response> on 2024-04-12 21:36:09.796259+00:00
[2024-04-12T21:36:13.864+0000] {standard_task_runner.py:60} INFO - Started process 1841 to run task
[2024-04-12T21:36:13.867+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'KingFX_ETL', 'Extract_Response', 'manual__2024-04-12T21:36:09.796259+00:00', '--job-id', '48', '--raw', '--subdir', 'DAGS_FOLDER/include/fx_dag.py', '--cfg-path', '/tmp/tmpfv2k4k82']
[2024-04-12T21:36:13.870+0000] {standard_task_runner.py:88} INFO - Job 48: Subtask Extract_Response
[2024-04-12T21:36:13.914+0000] {task_command.py:423} INFO - Running <TaskInstance: KingFX_ETL.Extract_Response manual__2024-04-12T21:36:09.796259+00:00 [running]> on host 65f2ce3f86e2
[2024-04-12T21:36:14.000+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='kresthill' AIRFLOW_CTX_DAG_ID='KingFX_ETL' AIRFLOW_CTX_TASK_ID='Extract_Response' AIRFLOW_CTX_EXECUTION_DATE='2024-04-12T21:36:09.796259+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-12T21:36:09.796259+00:00'
[2024-04-12T21:36:15.599+0000] {logging_mixin.py:188} INFO - {'code': 1, 'message': 'Bad credentials', 'documentation_url': 'https://xecdapi.xe.com/docs/v1/'}
[2024-04-12T21:36:15.601+0000] {python.py:202} INFO - Done. Returned value was: None
[2024-04-12T21:36:15.614+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=KingFX_ETL, task_id=Extract_Response, execution_date=20240412T213609, start_date=20240412T213613, end_date=20240412T213615
[2024-04-12T21:36:15.654+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
